### DISCUSSION TITLE
Pakage manager

### DISCUSSION THREAD

**Discussion by shivarm** [quarkiverse/quarkus-quinoa#518] [Q&A] [answered]
@ia3andy @melloware 

What is the version of package manager used by quinoa? like , npm, yarn...

**Comment by melloware** [comment]
It is controlled by a property in the application.propweties.  So users can use whatever version they want

**Reply by shivarm** [reply]
@melloware `quarkus.quinoa.package-manager-install.node-version=18.18.0` are you talking about this if so then my question is what npm version used by quinoa?

**Reply by melloware** [reply, marked as answer]
Isn't that this setting: https://docs.quarkiverse.io/quarkus-quinoa/dev/config-reference.html#quarkus-quinoa_quarkus.quinoa.package-manager-install.npm-version

> The NPM version to install and use. By default, the version is provided by NodeJS.

**Reply by shivarm** [reply]
Thanks
